### PR TITLE
chore: Use iab as cache prefix

### PR DESCRIPTION
**Pull request by microamp**
Found one minor discrepancy when diffing against the previous prod config.

### CODE REVIEW

**Review by kesara — CHANGES_REQUESTED** [review]

File: helm/local.py [hunk 1]
```diff
@@ -64,12 +64,12 @@
     "default": {
         "BACKEND": "django.core.cache.backends.memcached.PyMemcacheCache",
         "LOCATION": f"{MEMCACHED_HOST}:{MEMCACHED_PORT}",
-        "KEY_PREFIX": "ietf",
+        "KEY_PREFIX": "iab",
```
[the remark this answers is not in the document]
Comment (microamp) [reply helm/local.py:67]: Understood. Probably not worth a new release. I find `ietf` generic enough (for both sites). Will be revisited otherwise.

**Comment by rjsparks** [comment]
Why do you expect sharing a key prefix to be safe in this case? What's the other thing going into key generation that's guaranteed to avoid key collisions between the two wagtail instances?
See https://github.com/django/django/blob/ca5cd3e3e8e53f15e68ccd727ec8fe719cc48099/django/core/cache/backends/base.py#L32-L40 - I think it's pretty likely that wagtail itself would be generating the same prefixes for things it would want to cache? Or does wagtail go out of its way to ensure the prefix is unique based on the config? (If so, point to the code that does so?)

**Comment by rjsparks** [comment]
(It may not matter as long as you guarantee that the ietf.org and iab.org instances are not sharing an instance of memcached, but if we ever were to put these back on one big host...)

**Comment by kesara** [comment]
> (It may not matter as long as you guarantee that the ietf.org and iab.org instances are not sharing an instance of memcached, but if we ever were to put these back on one big host...)

Yeah, separation was the reason. Even on ietfa, the two sites did not share a single Memcached instance.
But if we think they might share a single Memcache db, we should use the Helm variable for that.

**Pull request closed without merging**